src/pyats/contrib/creators/topology.py

```diff
@@ -0,0 +1,836 @@
+import os

```

Comment:
add mode to not send any configuration at connection

Reply:
created disable-config